<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>571</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="sitesList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailAddress">
       <property name="text">
        <string>email address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>571</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="NewSiteName">
       <property name="text">
        <string>site name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NewEmailAddress">
       <property name="text">
        <string>email address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NewUsername">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NewPassword">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="getPasswordButton">
       <property name="text">
        <string>get password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteDataButton">
       <property name="text">
        <string>delete data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeDataButton">
       <property name="text">
        <string>change data</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkRandomPassword">
       <property name="text">
        <string>random password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuContents">
    <property name="title">
     <string>Contents</string>
    </property>
   </widget>
   <addaction name="menuContents"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
